src/libraries/System.Security.Cryptography.ProtectedData/src/System/Security/Cryptography/ProtectedData.cs

```diff
@@ -388,16 +390,16 @@ private static bool TryProtectOrUnprotect(
                             return false;
                         }
 
-                        new Span<byte>(outputBlob.pbData.ToPointer(), length).CopyTo(outputSpan);
+                        interopSpan.CopyTo(outputSpan);
                         bytesWritten = length;
                         outputData = null;
                         return true;
                     }
                     finally
                     {
-                        if (outputBlob.pbData != IntPtr.Zero)
+                        if (!interopSpan.IsEmpty)
                         {
-                            CryptographicOperations.ZeroMemory(new Span<byte>(outputBlob.pbData.ToPointer(), (int)outputBlob.cbData));
+                            interopSpan.Clear();
```

Comment:
This should still use ZeroMemory.
```suggestion
                            CryptographicOperations.ZeroMemory(interopSpan);
```

Reply:
It can't, because that doesn't work in netstandard.  I could put an `#if NETSTANDARD` block there, but since they both make it zero in the end, that complication didn't seem worth it, but I can do that if you feel strongly about it.

Reply:
Ah okay, I was looking at an incomplete diff between two of your commits, not `main`.

Reply:
I changed to it per one of @bartonjs suggestions, but then the build failed, and I was sad.

Reply:
In some other projects we have a back-compat implementation, e.g. for Microsoft.Bcl.Cryptography.

https://github.com/dotnet/runtime/blob/f6345d24b90ed79d919dc0066db913bdc060914a/src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/NetStandardShims.cs#L128-L134

You... could... introduce a similar Shim file, and only compile it an `ItemGroup` with `Condition=" '$(TargetFrameworkIdentifier)' != '.NETCoreApp' "`

Reply:
That's mostly because Microsoft.Bcl.Cryptography ingested a bunch of code that used it.

Just Clear here is... fine.

Reply:
It was there before the cert loader, but sure, I don't disagree the `Clear` is fine.